claragenomics/dl4atac/utils.py

```diff
@@ -8,57 +8,87 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-import sys
-import torch
+"""Utility functions."""
 import shutil
 import os
 import time
+
 import h5py
 import numpy as np
 from datetime import datetime
+from termcolor import colored
 import torch
 import torch.distributed as dist
-from termcolor import colored
+
 import yaml
 
+
 def myprint(msg, color=None, rank=0):
```

Comment:
@avantikalal , I'm fairly confident about this file, but it would be good for you to verify.